File: android-room-sample-master/app/src/javaDagger/java/com/emreeran/android/roomsample/di/ActivityModule.java
package com.emreeran.android.roomsample.di;

import com.emreeran.android.roomsample.MainActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

/**
 * Created by Emre Eran on 26.05.2018.
 */
@Module
abstract class ActivityModule {

    @ContributesAndroidInjector(modules = {MainFragmentBuildersModule.class})
    abstract MainActivity contributeMainActivity();
}

File: android-room-sample-master/app/src/kotlinDagger/java/com/emreeran/android/roomsample/di/ActivityModule.kt
package com.emreeran.android.roomsample.di

import com.emreeran.android.roomsample.MainActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

/**
 * Created by Emre Eran on 20.04.2018.
 */
@Module
abstract class ActivityModule {

    @ContributesAndroidInjector(modules = [(MainFragmentBuildersModule::class)])
    internal abstract fun contributeMainActivity(): MainActivity
}